app/models/article.rb

```diff
@@ -63,6 +65,10 @@ def article_date_cannot_be_in_the_future
     end
   end
 
+  def edit_summary
```

Comment:
this is needed to work with the paper_trail gem.  for some reason it needs to be a model method, not merely a model attribute to be recorded as meta data.  Thus the repetitive method is for this line...

`has_paper_trail :ignore => [:summary], :meta => { :comment  => :edit_summary }`